mh/constants.py

```diff
@@ -164,13 +165,41 @@ def make_binary_npc_greeters(is_jap=False):
     US_OFFSET = 0x180
     JP_OFFSET = 0x100
     offset = JP_OFFSET if is_jap else US_OFFSET
+
+
+    if is_jhen_active():
+        tool_shop_str = "Half-off sale!"
+        material_shop_str = "Half-off sale!"
+        event_quests_str = "The Festival of Fear\n"
+    else:
+        tool_shop_str = "Prices are normal."
+        material_shop_str = "Prices are normal."
+        event_quests_str = ""
+
+    event_quests_str += QUEST_EVENT_JUMP_FOUR_JAGGI[0:40].decode('ascii').rstrip('\x00') + '\n'
+    event_quests_str += QUEST_EVENT_BLOOD_SPORT[0:40].decode('ascii').rstrip('\x00') + '\n'
+    event_quests_str += QUEST_EVENT_MERCY_MISSION[0:40].decode('ascii').rstrip('\x00') + '\n'
+    event_quests_str += QUEST_EVENT_THE_PHANTOM_URAGAAN[0:40].decode('ascii').rstrip('\x00') + '\n'
+    event_quests_str += QUEST_EVENT_WORLD_EATER[0:40].decode('ascii').rstrip('\x00') + '\n'
+    event_quests_str += QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD[0:40].decode('ascii').rstrip('\x00') + '\n'
+    event_quests_str += QUEST_EVENT_GREEN_EGGS[0:40].decode('ascii').rstrip('\x00')
+
+    guildmaster_str = "To all hunters:\n"
+    guildmaster_str += "Loc Lac is in \"beta,\"\n"
+    guildmaster_str += "whatever that means. Also,\n"
+    guildmaster_str += "the Arena is currently under\n"
+    guildmaster_str += "construction. And now, a:\n"
+    guildmaster_str += "haiku: \"Patient warriors /\n"    
+    guildmaster_str += "returning home at long last /\n"
+    guildmaster_str += "now join hands to hunt!\""
```

Comment:
The following should work and get rid of the concatenation:
```python
guildmaster_str = (
    b'To all hunters:\n'
    b'Loc Lac is in "beta",\n'
    b'whatever that means. Also,\n'
    b'the Arena is currently under\n'
    b'construction. And now, a:\n'
    b'haiku: "Patient warriors /\n'
    b'returning home at long last /\n'
    b'now join hands to hunt!"'
)
```

Reply:
Done